src/helpers/articles.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
```

Comment:
Here also. I wouldn't want you disabling eslint rules, except you have a good reason to.